app/javascript/src/QuickStarts/templates/createMethod.yaml

```diff
@@ -0,0 +1,101 @@
+    
+    To learn more about API methods, see the [3scale documentation](https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.10/html-single/admin_portal_guide/index#designating-methods-and-adding-metrics-for-capturing-usage-details_methods-and-metrics-for-products-and-backends).
+
+    You can designate methods for products and backends. Adding a method to a product or backend means that you are designating a method in your API for which you want to capture individual usage details. An application plan provides the ability to set a limit for each method that you add to a product or backend.
+
+    This provides finer granularity for tracking API use. Reporting traffic to a method automatically increases counters for the method and for the Hits metric.
+  tasks:
+    - title: Add methods to backend
+      description: |-
+        For backends, adding methods enables setting limits and pricing rules in the application plans for any product that bundles that backend.
+
+        ### Prerequisites
+
+        - A backend. To create one, see Create a backend.
+        
+        ### Procedure
+
+        1. Navigate to <strong>[Your\_backend\_name] > Methods and Metrics</strong>.
+        1. Click <strong>New method</strong>.
+        1. On the _New method_ page, fill in the details to create the method.
+        
+          Important fields:
+              - _Friendly name_: A short description of the method. This name is displayed in different sections of the 3scale Admin Portal.
+              - _System name_: The name of the method in your API to use to report the usage through the 3scale Service Management API. The system name must be unique.
+              > <strong>Tip:</strong> Be careful with changing the system name of the methods or deleting them. These changes can break your already deployed 3scale integration if there are mapping rules pointing to the previous system name of the method.
+        1. Click <strong>Create method</strong>
+
+      # optional - the task's Check your work module
+      review:
+        instructions: |-
+          - Did you create a unique system name for the method?
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+      # optional - the task's success and failure messages
+      summary:
+        success: Shows a success message in the task header
+        failed: Shows a failed message in the task header
+    - title: Add methods to product
+      description: |-
+        For products, adding methods enables setting limits and pricing rules in the product’s application plans. The procedure for adding a method to a product is similar to adding a method to a backend.
+
+        ### Prerequisites
+
+        - A product. To create one, see Create a product.
+        - An application plan. To create one, see Create an application plan.
```

Comment:
see "Create an application plan" is a link, which link to 'createApplicationPlan.yaml'